### PR TITLE
193 resolve docs issues

### PR DESCRIPTION
**Pull request by hulto**
Adds golem to toc and fixes doc typos.

### CODE REVIEW

File: docs/_docs/user-guide/eldritch.md [hunk 1]
```diff
@@ -223,10 +228,12 @@ The <b>pivot.arp_scan</b> method is being proposed to allow users to enumerate h
 The <b>pivot.port_forward</b> method is being proposed to providde socat like functionality by forwarding traffic from a port on a local machine to a port on a different machine allowing traffic to be relayed.
 
 ### pivot.ncat
-`pivot.ncat(address: str, port: int, data: str, str: portocol ) -> str`
+`pivot.ncat(address: str, port: int, data: str, portocol: str ) -> str`
```
Comment (Milkshak3s) [inline docs/_docs/user-guide/eldritch.md:231]: portocol, or protocol?

**Review by Milkshak3s — CHANGES_REQUESTED** [review]

File: docs/_docs/user-guide/eldritch.md (hunk 1 above)
Comment (Milkshak3s) [reply docs/_docs/user-guide/eldritch.md:231]: Or is this intentional :) 

https://github.com/KCarretto/realm/blob/995d1beca806d43465e58ca1767c4b7a950b538b/implants/eldritch/src/pivot.rs#LL83C80-L83C80

https://github.com/KCarretto/realm/blob/995d1beca806d43465e58ca1767c4b7a950b538b/implants/eldritch/src/pivot.rs#L78

https://github.com/KCarretto/realm/blob/995d1beca806d43465e58ca1767c4b7a950b538b/implants/eldritch/src/pivot/ncat_impl.rs#L57

Comment (hulto) [reply docs/_docs/user-guide/eldritch.md:231]: 😅 just wish I knew how to spell.

**Review by Milkshak3s — APPROVED** [review]
My old requested changes aren't valid, unless we're also fixing the spelling in code:

https://github.com/KCarretto/realm/blob/995d1beca806d43465e58ca1767c4b7a950b538b/implants/eldritch/src/pivot.rs#L78

**Comment by hulto** [comment]
Yep haven't implemented those changes yet.

**Pull request merged**